with 1:
	try:
		raise NameError('YourNameIncorrect')
	except:
		raise NameError('YourNameIncorrect')
	try:
		raise NameError('YourNameIncorrect')
	except:
		raise NameError('YourNameIncorrect')
	print 1
